stack-8.8.yaml

```diff
@@ -4,6 +4,8 @@ extra-deps:
 - source-constraints-0.0.1@sha256:abdda6ca3d942c81d61ed4374dbc0eee21e9c074da3f17b25d434c2d55df2acc,2979
 - tasty-1.3.1@sha256:01e35c97f7ee5ccbc28f21debea02a38cd010d53b4c3087f5677c5d06617a507,2520
 - tasty-mgolden-0.0.1@sha256:07fadb592cde353c386c5f38cff75539a5675e9350776ed6f4f0a70247427a78,4214
+- git: https://github.com/lspitzner/brittany.git
+  commit: 55f84c329414814c718a81f590fece76c3f91ad1
```

Comment:
Too bad there is no release for us, this forces all devtools users (?) to carry the brittany reference.

Reply:
This particular release has patches that allow Brittany to work with code that has pattern synonyms; we could fall back to the official release and just tell brittany not to bother formatting that kind of code through an inline comment directive.

Reply:
I have removed this brittany reference.